dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -274,12 +294,71 @@ def __init__(self,
 
             super(_DALIDatasetV2, self).__init__(self._as_variant_tensor())
 
-        def _check_output_dtypes(self, output_dtypes):
-            """Check whether output_dtypes is instance of tf.DType or tuple of tf.DType"""
-            if isinstance(output_dtypes, tf.DType):
+        def _parse_inputs(self, input_datasets, input_names, input_layouts, input_batch):
```

Comment:
The name suggests that this function processes some data stream or text - none of this happens. `_setup_inputs` or `_validate_inputs`?

Reply:
Let it be setup.